browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -287,12 +287,6 @@ void SpeedreaderTabHelper::ShowBubble(bool is_bubble_speedreader) {
 }
 
 void SpeedreaderTabHelper::HideBubble() {
-  Browser* browser = chrome::FindBrowserWithWebContents(web_contents());
```

Comment:
It used from OnWebContentsDestroyed which called form web_contents destructor so we can't find any browser by this web_contents